libc/string_lenchr.c

```diff
@@ -0,0 +1,137 @@
+/*
+ * Phoenix-RTOS
+ *
+ * POSIX functions tests
+ * Tested functions:
+ * 	strlen()
+ * 	strnlen()
```

Comment:
Ok, final proposition, lets use `TESTED` it's done in stdio_file.c

```suggestion
 * POSIX.1-2017 standard library functions tests
 * HEADER:
 *       - string.h
 * TESTED:
 * 	 - strlen()
 * 	 - strnlen()
```